source/Table/Table.js

```diff
@@ -443,23 +445,24 @@ export default class Table extends PureComponent {
 
     const style = this._cachedColumnStyles[columnIndex];
 
-    const title = typeof renderedCell === 'string' ? renderedCell : null;
+    const columnTitle = title ? title : (typeof renderedCell === 'string' ? renderedCell : null);
 
-    const a11yProps = {
+    const columnProps = {
       role: 'gridcell',
+      'aria-describedby': id || undefined,
+      key: "Row" + rowIndex + "-" + "Col" + columnIndex,
+      className: cn('ReactVirtualized__Table__rowColumn', className),
+      style: style,
+      title: columnTitle,
     };
-
-    if (id) {
-      a11yProps['aria-describedby'] = id;
-    }
-
+    
+    // NOTE: is important to NOT mixin inline property and spread properties,
+    // since internally REACT clone property object to adds inline property
+    // and slow down performances
```

Comment:
The wording of this comment (and the one below) confuse me. I don't think this has anything to do with "React internals" (unless I'm misunderstanding something). I believe this change is being made to avoid the overhead added by the [`babel-runtime` `extends` method](https://unpkg.com/babel-runtime@6.26.0/helpers/extends).

Before this change:
```js
return _react2.default.createElement(
  'div',
  (0, _extends3.default)({}, a11yProps, {
    key: 'Row' + rowIndex + '-Col' + columnIndex,
    className: (0, _classnames2.default)('ReactVirtualized__Table__rowColumn', className),
    style: style,
    title: title }),
  renderedCell
);
```

After this change:
```js
return (
  <div
    {...columnProps}
   >
    {renderedCell}
  </div>
);
```

Reply:
That being said, I'm happy to update the wording of the comment myself, so don't worry about it. 😄